test/nexus-integration.py

```diff
@@ -41,10 +41,11 @@ def make_input(version, **kwargs):
     resconfig = {
         'source': {
             'name': 'somedemo',
-            'username': 'admin',
-            'password': 'admin123',
+            'name_must_match': False,
```

Comment:
New feature introduced in 0.7.0 but not taken into account in nexus tests. Hence majority of tests are failing without it.